### PR TITLE
feat(dh): use watt-paginator for wholesale

### PR DESCRIPTION
**Pull request by ManBearTM**

### CODE REVIEW

File: libs/ui-watt/src/lib/components/table/watt-table.component.scss [hunk 1]
```diff
@@ -23,6 +23,7 @@ watt-table .mat-table {
   grid-auto-rows: max-content;
   height: 100%;
   overflow: auto;
+  isolation: isolate;
```
Comment (frontend-specialisten) [inline libs/ui-watt/src/lib/components/table/watt-table.component.scss:26]: Praise: Cool, I didn't know this property

File: libs/dh/wholesale/feature-calculation-steps/src/lib/time-series-points/dh-wholesale-time-series-points.component.scss [hunk 2]
```diff
@@ -19,8 +19,18 @@
   width: 100%;
 }
 
+watt-card {
```
Comment (frontend-specialisten) [inline libs/dh/wholesale/feature-calculation-steps/src/lib/time-series-points/dh-wholesale-time-series-points.component.scss:22]: Question: could we do some context-specific styling with ex. `:host-context('something something')` for the card, so we don't have to do this in multiple places?

**Review by frontend-specialisten — APPROVED** [review]

File: libs/ui-watt/src/lib/components/table/watt-table.component.scss (hunk 1 above)
Comment (ManBearTM) [reply libs/ui-watt/src/lib/components/table/watt-table.component.scss:26]: Neither did I 😄 Much better than the old hacks to force a new stacking context!

File: libs/dh/wholesale/feature-calculation-steps/src/lib/time-series-points/dh-wholesale-time-series-points.component.scss (hunk 2 above)
Comment (ManBearTM) [reply libs/dh/wholesale/feature-calculation-steps/src/lib/time-series-points/dh-wholesale-time-series-points.component.scss:22]: Oh cool, wasn't aware of `:host-context`, good idea! Will check it out 🙂

**Pull request closed without merging**